rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -834,16 +834,25 @@ def _create_demographic_fields(self, data):
             logger.info("creating demographic fields ..")
             logger.info("d = %s" % d)
             registry_obj = Registry.objects.get(code=d["registry"])
-            group_obj, created = Group.objects.get_or_create(name=d["group"])
-            if created:
-                logger.info("created Group %s" % group_obj)
-                group_obj.save()
-
+            groups = []
```

Comment:
Just to make it more visible on what's different between the 2 cases:

```python
group_names = d.get('groups', []])
# Support for importing data in the old format. We used to have demographic fields assigned to only one group in the past.
if 'group' in d:
    group_names = [d['group']]
for group_name in group_names:
    ...
```

Reply:
Updated.